src/NServiceBus.Core/Pipeline/Outgoing/InspectOutgoingMessageBehavior.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.Pipeline;
+
+    class InspectOutgoingMessageBehavior : Behavior<IOutgoingPhysicalMessageContext>
```

Comment:
The fix it to use a new header for the "Time that the message being audited was originally sent"  value but that's hard when it comes to backwards compat (still doable)